museum/sql/testImports/baseObject.sql

```diff
@@ -0,0 +1,16 @@
+# find at least 2 representative values for each field
+(SELECT PrimaryKey_Object_Table FROM Objects WHERE ItemType IS NOT NULL LIMIT 2)
```

Comment:
Keeping it separate means we can ensure that we're getting the same sample records each time by hard coding their primary key in the query. This means we can re-run the script and know that the sample records will come out in the same order.